tests/test_importjob.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+from django.core.files.base import ContentFile
+from django.test import TestCase
+
+from import_export_celery.models.importjob import ImportJob
+
+
+class ImportJobTestCases(TestCase):
+
+## 9-10-2023 uitgezet vanwege error op permissies op de media foler -> lokaal draait alles nu prima
+
+     def test_delete_file_on_job_delete(self):
```

Comment:
Deze test moet of weg, of opgelost worden.

Reply:
weggegooid